From b40ff016ece1350306acf483dd3552aadad29dd3 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 12 Oct 2018 10:17:38 +0300
Subject: [PATCH] version-hooks: simplify the version creation

It turns out that git describe has a '--dirty' flag which checks
the working tree and appends the <mark> (-dirty by default) if the
working tree is dirty.
Make use of it and remove the 'git status --porcelain' call.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/package/version-hooks/version-hooks.mk b/package/version-hooks/version-hooks.mk
index 4451a21..aa41bd9 100644
--- a/package/version-hooks/version-hooks.mk
+++ b/package/version-hooks/version-hooks.mk
@@ -1,10 +1,7 @@
 define HL_VERSION_GEN
 	if [ -n $(SRCDIR) ]; then \
 		pushd $(SRCDIR); \
-		echo -n "-`git describe 2>/dev/null`" > $(@D)/localversion-habanalabs; \
-		if [ -n "`git status --porcelain 2>&1`" ]; then \
-			printf '%s' -dirty >> $(@D)/localversion-habanalabs; \
-		fi; \
+		echo -n "-`git describe --dirty 2>/dev/null`" > $(@D)/localversion-habanalabs; \
 		popd; \
 	fi
 endef
